src/app/containers/PageHandlers/withLoading/__snapshots__/index.test.jsx.snap

```diff
@@ -177,50 +177,78 @@ exports[`withLoading HOC and the loading prop set to true should return the load
 }
 
 @media (min-width:37.5rem) and (max-width:62.9375rem) {
-    .emotion-0 {
+    .emotion-4 {
       width: initial;
       margin: 0;
-      grid-template-columns: repeat(5,1fr);
-      grid-column-end: span 5;
+      grid-template-columns: repeat(6,1fr);
+      grid-column-end: span 6;
       padding: 0 1rem;
       grid-column-start: 1;
     }
 }
 
 @media (min-width:63rem) and (max-width:79.9375rem) {
-    .emotion-0 {
+    .emotion-4 {
       width: initial;
       margin: 0;
-      grid-template-columns: repeat(5,1fr);
-      grid-column-end: span 5;
+      grid-template-columns: repeat(6,1fr);
+      grid-column-end: span 6;
       grid-column-start: 2;
     }
 }
 
 @media (min-width:80rem) {
-    .emotion-0 {
+    .emotion-4 {
       width: initial;
       margin: 0;
-      grid-template-columns: repeat(10,1fr);
-      grid-column-end: span 10;
+      grid-template-columns: repeat(12,1fr);
+      grid-column-end: span 12;
       grid-column-start: 5;
     }
 }
 }
 
+.emotion-2 {
+  outline: 0;
+}
+
+.emotion-0 {
+  -webkit-clip-path: inset(100%);
+  clip-path: inset(100%);
+  -webkit-clip: rect(1px,1px,1px,1px);
+  clip: rect(1px,1px,1px,1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  margin: 0;
+}
+
 <div>
   <main
-    class="emotion-5 emotion-6"
+    class="emotion-9 emotion-10"
     role="main"
   >
     <div
-      class="emotion-2 emotion-3 emotion-1"
+      class="emotion-6 emotion-7 emotion-5"
       dir="ltr"
     >
       <div
-        class="emotion-0 emotion-1"
+        class="emotion-4 emotion-5"
         dir="ltr"
-      />
+      >
+        <div
```

Comment:
Not sure what all these divs are for... but I think inorder for this to work with all our supported AT, we are going to need to move focus to the actual element with the content e.g the span/p and instead have the tabindex on this element.